.github/workflows/kafka_demo_test.yml

```diff
@@ -0,0 +1,177 @@
+name: Kafka Demo Test 
+
+on:
+  pull_request:
+    branches: 
+      - main
+  workflow_call:
+    inputs:
+      sidecar-registry:
+        description: "The registry to get the image from"
+        required: true
+        default: "mcr.microsoft.com"
+        type: string
+      key-release-image:
+        description: "The image of the SKR sidecar to use"
+        required: true
+        default: "aci/skr:2.7"
+        type: string
+      consumer-image: 
+        description: "Consumer Image"
+        required: true
+        type: string
+      producer-image: 
+        description: "Producer Image"
+        required: true
+        type: string
+      skr-kafka-demo-topic:
+        description: "Kafka topic"
+        required: true
+        default: "kafka-demo-topic"
+        type: string  
+  workflow_dispatch:
+    inputs:
+      sidecar-registry:
+        description: "The registry to get the image from"
+        required: true
+        type: choice
+        default: mcr.microsoft.com
+        options: 
+          - mcr.microsoft.com
+          - confidentialsidecars.azurecr.io
+      key-release-image:
+        description: "The image of the SKR sidecar to use"
+        required: true
+        default: "aci/skr:2.7"
+        type: string
+      consumer-image: 
+        description: "Consumer Image"
+        default: acc/samples/kafka/consumer:1.0
+        required: true
+        type: string
+      producer-image: 
+        description: "Producer Image"
+        default: acc/samples/kafka/producer:1.0
+        required: true
+        type: string
+  merge_group:
+    branches:
+      - main
+
+jobs:
+  build-image:
+    name: Build Image 
+    runs-on: ubuntu-latest
+    if: ${{ inputs.consumer-image }} == '' || ${{ inputs.producer-image }} == ''
+    outputs:
+      consumer-image: ${{ steps.tag-image.outputs.consumer-image }}
+      producer-image: ${{ steps.tag-image.outputs.producer-image }}
+    steps: 
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: main
+
+      - name: Log in to Azure Container Registry
+        uses: azure/docker-login@v1
+        with:
+          login-server: ${{ vars.AZURE_REGISTRY_URLL }}
```

Comment:
why do all of these have an extra letter at the end?

Reply:
build image part is already removed.